src/wire/dhcpv4.rs

```diff
@@ -55,6 +57,58 @@ impl MessageType {
     }
 }
 
+/// A buffer for DHCP options.
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+pub struct DhcpOptionsRepr<T> {
```

Comment:
this is not a "repr". Reprs are Rust structs with everything neatly parsed into fields, this wraps an unparsed buffer. Perhaps name it "DhcpOptionsBuffer"?

Reply:
Agreed. Renamed to `DhcpOptionsBuffer`.